2023-01-09T04:58:46.788691203Z I0109 04:58:46.788563       1 readyz.go:155] Listening on 0.0.0.0:9980
2023-01-09T04:58:50.278432145Z I0109 04:58:50.278395       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:58:50.281272386Z 2023/01/09 04:58:50 WARNING: [core] [Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:58:50.281272386Z   "Addr": "localhost:2379",
2023-01-09T04:58:50.281272386Z   "ServerName": "localhost",
2023-01-09T04:58:50.281272386Z   "Attributes": null,
2023-01-09T04:58:50.281272386Z   "BalancerAttributes": null,
2023-01-09T04:58:50.281272386Z   "Type": 0,
2023-01-09T04:58:50.281272386Z   "Metadata": null
2023-01-09T04:58:50.281272386Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:58:51.282769597Z 2023/01/09 04:58:51 WARNING: [core] [Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:58:51.282769597Z   "Addr": "localhost:2379",
2023-01-09T04:58:51.282769597Z   "ServerName": "localhost",
2023-01-09T04:58:51.282769597Z   "Attributes": null,
2023-01-09T04:58:51.282769597Z   "BalancerAttributes": null,
2023-01-09T04:58:51.282769597Z   "Type": 0,
2023-01-09T04:58:51.282769597Z   "Metadata": null
2023-01-09T04:58:51.282769597Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:58:52.279808508Z W0109 04:58:52.279769       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T04:58:54.280084432Z I0109 04:58:54.280042       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:58:54.281045922Z 2023/01/09 04:58:54 WARNING: [core] [Channel #3 SubChannel #4] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:58:54.281045922Z   "Addr": "localhost:2379",
2023-01-09T04:58:54.281045922Z   "ServerName": "localhost",
2023-01-09T04:58:54.281045922Z   "Attributes": null,
2023-01-09T04:58:54.281045922Z   "BalancerAttributes": null,
2023-01-09T04:58:54.281045922Z   "Type": 0,
2023-01-09T04:58:54.281045922Z   "Metadata": null
2023-01-09T04:58:54.281045922Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:58:55.279472073Z I0109 04:58:55.279430       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:58:55.280469737Z 2023/01/09 04:58:55 WARNING: [core] [Channel #5 SubChannel #6] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:58:55.280469737Z   "Addr": "localhost:2379",
2023-01-09T04:58:55.280469737Z   "ServerName": "localhost",
2023-01-09T04:58:55.280469737Z   "Attributes": null,
2023-01-09T04:58:55.280469737Z   "BalancerAttributes": null,
2023-01-09T04:58:55.280469737Z   "Type": 0,
2023-01-09T04:58:55.280469737Z   "Metadata": null
2023-01-09T04:58:55.280469737Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:58:55.281901771Z 2023/01/09 04:58:55 WARNING: [core] [Channel #3 SubChannel #4] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:58:55.281901771Z   "Addr": "localhost:2379",
2023-01-09T04:58:55.281901771Z   "ServerName": "localhost",
2023-01-09T04:58:55.281901771Z   "Attributes": null,
2023-01-09T04:58:55.281901771Z   "BalancerAttributes": null,
2023-01-09T04:58:55.281901771Z   "Type": 0,
2023-01-09T04:58:55.281901771Z   "Metadata": null
2023-01-09T04:58:55.281901771Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:58:56.280518581Z W0109 04:58:56.280476       1 etcdcli_pool.go:73] could not create a new cached client after 1 tries, trying again. Err: context deadline exceeded
2023-01-09T04:58:56.282043940Z 2023/01/09 04:58:56 WARNING: [core] [Channel #5 SubChannel #6] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:58:56.282043940Z   "Addr": "localhost:2379",
2023-01-09T04:58:56.282043940Z   "ServerName": "localhost",
2023-01-09T04:58:56.282043940Z   "Attributes": null,
2023-01-09T04:58:56.282043940Z   "BalancerAttributes": null,
2023-01-09T04:58:56.282043940Z   "Type": 0,
2023-01-09T04:58:56.282043940Z   "Metadata": null
2023-01-09T04:58:56.282043940Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:58:57.280765687Z W0109 04:58:57.280731       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T04:58:59.281107123Z I0109 04:58:59.281065       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:58:59.282194005Z 2023/01/09 04:58:59 WARNING: [core] [Channel #7 SubChannel #8] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:58:59.282194005Z   "Addr": "localhost:2379",
2023-01-09T04:58:59.282194005Z   "ServerName": "localhost",
2023-01-09T04:58:59.282194005Z   "Attributes": null,
2023-01-09T04:58:59.282194005Z   "BalancerAttributes": null,
2023-01-09T04:58:59.282194005Z   "Type": 0,
2023-01-09T04:58:59.282194005Z   "Metadata": null
2023-01-09T04:58:59.282194005Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:00.280571535Z I0109 04:59:00.280531       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:59:00.280571535Z I0109 04:59:00.280546       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:59:00.281376368Z 2023/01/09 04:59:00 WARNING: [core] [Channel #10 SubChannel #11] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:00.281376368Z   "Addr": "localhost:2379",
2023-01-09T04:59:00.281376368Z   "ServerName": "localhost",
2023-01-09T04:59:00.281376368Z   "Attributes": null,
2023-01-09T04:59:00.281376368Z   "BalancerAttributes": null,
2023-01-09T04:59:00.281376368Z   "Type": 0,
2023-01-09T04:59:00.281376368Z   "Metadata": null
2023-01-09T04:59:00.281376368Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:00.281404660Z 2023/01/09 04:59:00 WARNING: [core] [Channel #9 SubChannel #12] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:00.281404660Z   "Addr": "localhost:2379",
2023-01-09T04:59:00.281404660Z   "ServerName": "localhost",
2023-01-09T04:59:00.281404660Z   "Attributes": null,
2023-01-09T04:59:00.281404660Z   "BalancerAttributes": null,
2023-01-09T04:59:00.281404660Z   "Type": 0,
2023-01-09T04:59:00.281404660Z   "Metadata": null
2023-01-09T04:59:00.281404660Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:00.282844259Z 2023/01/09 04:59:00 WARNING: [core] [Channel #7 SubChannel #8] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:00.282844259Z   "Addr": "localhost:2379",
2023-01-09T04:59:00.282844259Z   "ServerName": "localhost",
2023-01-09T04:59:00.282844259Z   "Attributes": null,
2023-01-09T04:59:00.282844259Z   "BalancerAttributes": null,
2023-01-09T04:59:00.282844259Z   "Type": 0,
2023-01-09T04:59:00.282844259Z   "Metadata": null
2023-01-09T04:59:00.282844259Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:01.281944775Z W0109 04:59:01.281907       1 etcdcli_pool.go:73] could not create a new cached client after 1 tries, trying again. Err: context deadline exceeded
2023-01-09T04:59:01.282406313Z 2023/01/09 04:59:01 WARNING: [core] [Channel #9 SubChannel #12] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:01.282406313Z   "Addr": "localhost:2379",
2023-01-09T04:59:01.282406313Z   "ServerName": "localhost",
2023-01-09T04:59:01.282406313Z   "Attributes": null,
2023-01-09T04:59:01.282406313Z   "BalancerAttributes": null,
2023-01-09T04:59:01.282406313Z   "Type": 0,
2023-01-09T04:59:01.282406313Z   "Metadata": null
2023-01-09T04:59:01.282406313Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:01.282436197Z 2023/01/09 04:59:01 WARNING: [core] [Channel #10 SubChannel #11] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:01.282436197Z   "Addr": "localhost:2379",
2023-01-09T04:59:01.282436197Z   "ServerName": "localhost",
2023-01-09T04:59:01.282436197Z   "Attributes": null,
2023-01-09T04:59:01.282436197Z   "BalancerAttributes": null,
2023-01-09T04:59:01.282436197Z   "Type": 0,
2023-01-09T04:59:01.282436197Z   "Metadata": null
2023-01-09T04:59:01.282436197Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:02.281691309Z W0109 04:59:02.281649       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T04:59:02.281691309Z W0109 04:59:02.281648       1 etcdcli_pool.go:73] could not create a new cached client after 2 tries, trying again. Err: context deadline exceeded
2023-01-09T04:59:03.386511999Z I0109 04:59:03.386467       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:59:03.388461095Z 2023/01/09 04:59:03 WARNING: [core] [Channel #13 SubChannel #14] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:03.388461095Z   "Addr": "localhost:2379",
2023-01-09T04:59:03.388461095Z   "ServerName": "localhost",
2023-01-09T04:59:03.388461095Z   "Attributes": null,
2023-01-09T04:59:03.388461095Z   "BalancerAttributes": null,
2023-01-09T04:59:03.388461095Z   "Type": 0,
2023-01-09T04:59:03.388461095Z   "Metadata": null
2023-01-09T04:59:03.388461095Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:04.282168086Z I0109 04:59:04.282110       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:59:04.283236100Z 2023/01/09 04:59:04 WARNING: [core] [Channel #15 SubChannel #16] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:04.283236100Z   "Addr": "localhost:2379",
2023-01-09T04:59:04.283236100Z   "ServerName": "localhost",
2023-01-09T04:59:04.283236100Z   "Attributes": null,
2023-01-09T04:59:04.283236100Z   "BalancerAttributes": null,
2023-01-09T04:59:04.283236100Z   "Type": 0,
2023-01-09T04:59:04.283236100Z   "Metadata": null
2023-01-09T04:59:04.283236100Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:04.390180540Z 2023/01/09 04:59:04 WARNING: [core] [Channel #13 SubChannel #14] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:04.390180540Z   "Addr": "localhost:2379",
2023-01-09T04:59:04.390180540Z   "ServerName": "localhost",
2023-01-09T04:59:04.390180540Z   "Attributes": null,
2023-01-09T04:59:04.390180540Z   "BalancerAttributes": null,
2023-01-09T04:59:04.390180540Z   "Type": 0,
2023-01-09T04:59:04.390180540Z   "Metadata": null
2023-01-09T04:59:04.390180540Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:05.281452815Z I0109 04:59:05.281418       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:59:05.281979132Z I0109 04:59:05.281951       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:59:05.282494424Z 2023/01/09 04:59:05 WARNING: [core] [Channel #17 SubChannel #18] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:05.282494424Z   "Addr": "localhost:2379",
2023-01-09T04:59:05.282494424Z   "ServerName": "localhost",
2023-01-09T04:59:05.282494424Z   "Attributes": null,
2023-01-09T04:59:05.282494424Z   "BalancerAttributes": null,
2023-01-09T04:59:05.282494424Z   "Type": 0,
2023-01-09T04:59:05.282494424Z   "Metadata": null
2023-01-09T04:59:05.282494424Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:05.282910960Z 2023/01/09 04:59:05 WARNING: [core] [Channel #19 SubChannel #20] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:05.282910960Z   "Addr": "localhost:2379",
2023-01-09T04:59:05.282910960Z   "ServerName": "localhost",
2023-01-09T04:59:05.282910960Z   "Attributes": null,
2023-01-09T04:59:05.282910960Z   "BalancerAttributes": null,
2023-01-09T04:59:05.282910960Z   "Type": 0,
2023-01-09T04:59:05.282910960Z   "Metadata": null
2023-01-09T04:59:05.282910960Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:05.284421183Z 2023/01/09 04:59:05 WARNING: [core] [Channel #15 SubChannel #16] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:05.284421183Z   "Addr": "localhost:2379",
2023-01-09T04:59:05.284421183Z   "ServerName": "localhost",
2023-01-09T04:59:05.284421183Z   "Attributes": null,
2023-01-09T04:59:05.284421183Z   "BalancerAttributes": null,
2023-01-09T04:59:05.284421183Z   "Type": 0,
2023-01-09T04:59:05.284421183Z   "Metadata": null
2023-01-09T04:59:05.284421183Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:05.387672445Z W0109 04:59:05.387638       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T04:59:06.283207654Z W0109 04:59:06.283167       1 etcdcli_pool.go:73] could not create a new cached client after 1 tries, trying again. Err: context deadline exceeded
2023-01-09T04:59:06.283603016Z 2023/01/09 04:59:06 WARNING: [core] [Channel #17 SubChannel #18] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:06.283603016Z   "Addr": "localhost:2379",
2023-01-09T04:59:06.283603016Z   "ServerName": "localhost",
2023-01-09T04:59:06.283603016Z   "Attributes": null,
2023-01-09T04:59:06.283603016Z   "BalancerAttributes": null,
2023-01-09T04:59:06.283603016Z   "Type": 0,
2023-01-09T04:59:06.283603016Z   "Metadata": null
2023-01-09T04:59:06.283603016Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:06.283631056Z 2023/01/09 04:59:06 WARNING: [core] [Channel #19 SubChannel #20] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:06.283631056Z   "Addr": "localhost:2379",
2023-01-09T04:59:06.283631056Z   "ServerName": "localhost",
2023-01-09T04:59:06.283631056Z   "Attributes": null,
2023-01-09T04:59:06.283631056Z   "BalancerAttributes": null,
2023-01-09T04:59:06.283631056Z   "Type": 0,
2023-01-09T04:59:06.283631056Z   "Metadata": null
2023-01-09T04:59:06.283631056Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:07.282786846Z W0109 04:59:07.282747       1 etcdcli_pool.go:73] could not create a new cached client after 0 tries, trying again. Err: context deadline exceeded
2023-01-09T04:59:07.282786846Z W0109 04:59:07.282747       1 etcdcli_pool.go:73] could not create a new cached client after 2 tries, trying again. Err: context deadline exceeded
2023-01-09T04:59:07.388008842Z I0109 04:59:07.387944       1 etcdcli_pool.go:70] creating a new cached client
2023-01-09T04:59:07.389379010Z 2023/01/09 04:59:07 WARNING: [core] [Channel #21 SubChannel #22] grpc: addrConn.createTransport failed to connect to {
2023-01-09T04:59:07.389379010Z   "Addr": "localhost:2379",
2023-01-09T04:59:07.389379010Z   "ServerName": "localhost",
2023-01-09T04:59:07.389379010Z   "Attributes": null,
2023-01-09T04:59:07.389379010Z   "BalancerAttributes": null,
2023-01-09T04:59:07.389379010Z   "Type": 0,
2023-01-09T04:59:07.389379010Z   "Metadata": null
2023-01-09T04:59:07.389379010Z }. Err: connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused"
2023-01-09T04:59:10.283427330Z I0109 04:59:10.283382       1 etcdcli_pool.go:70] creating a new cached client
